src/main/scala/fly/play/jiraExceptionProcessor/ReportingSupervisorStrategy.scala

```diff
@@ -42,15 +35,29 @@ class ReportingStrategyWrapper(wrapped: SupervisorStrategy, comment: String) ext
 
   private def reportException(context: ActorContext, exception: Throwable) = {
     val configuration = Configuration(context.system.settings.config)
-    val wsClient = {
-      val environment = Environment.simple(mode = Mode.Prod)
-      val wsConfig = new WSConfigParser(configuration, environment).parse
-      val ningWsConfig = new NingWSClientConfigParser(wsConfig, configuration, environment).parse
-      NingWSClient(ningWsConfig)
+
+    def withWsClient[T](code: WSClient => T): T = {
+      val wsClient = {
+        val environment = Environment.simple(mode = Mode.Prod)
+
+        val parser = new WSConfigParser(configuration, environment)
+        val config = AhcWSClientConfig(wsClientConfig = parser.parse())
+        val builder = new AhcConfigBuilder(config)
+        val ahcConfig = builder.configure().build()
+
+        implicit val system = ActorSystem("jiraExceptionProcessor")
```

Comment:
@EECOLOR If we use context.system and there is for some reason something wrong with it, the error won't be reported anymore. For all we know it might have already received a PoisonPill directly after the failure, although something like this is not very likely.

So I prefer that is uses its own ActorSystem to report the error to keep it a self contained functionality.